packages/store/tests/default-states.spec.ts

```diff
@@ -35,11 +35,11 @@ describe('Reusable States', () => {
     const store = TestBed.get(Store);
 
     let stateValue = store.selectSnapshot(FooState);
-    expect(stateValue).toEqual([4, 5, 6]);
+    expect(stateValue).toEqual([1, 2, 3]);
```

Comment:
Why did the expectation change? 
It looks like the FooState default is now overriding the provided default state.

Reply:
Yes! It turns out we laid a bug initially! When the storage plugin saved the state and started using it, then the life cycle of these states was no longer re-used! And in fact, the life cycle should not be re-used only for hmr.